**<font color='red' size='8px'>Question 1</font>**

The General Social Survey asked a random sample of 1,390 Americans the following question: “On the whole, do you think it should or should not be the government’s responsibility to promote equality between men and women?” 82% of the respondents said it “should be”. At a 95% confidence level, this sample has 2% margin of error. Based on this information, determine if the following statements are true or false, and explain your reasoning.

**<font color='green' size=6px>Answer 👇:**</font>


**<font color='orange' size='6px'>A)</font>**  

- **We are 95% confident that between 80% and 84% of Americans in this sample think it’s the government’s responsibility to promote equality between men and women.**
  - <font color='red'>**FALSE**</font>
    - <font color='blue'>**95% of the confidence we have is about confidence intervals. About the intervals that we make with the sample size of 1390, not about the Americans who are in this sample.**</font>


**<font color='orange' size='6px'>B)</font>**  
- **We are 95% confident that between 80% and 84% of all Americans think it’s the government’s responsibility to promote equality between men and women.**
  - <font color='red'>**FALSE**</font>
    - <font color='blue'>**This is cannot be used for entire population. We are not talking about 80% to 84% of Americans. When we say 95% confidence interval, it means 95% of these intervals (80% to 84%) would include the true population proportion of Americans.**</font>

**<font color='orange' size='6px'>C)</font>**  
- **If we considered many random samples of 1,390 Americans, and we calculated 95% confidence intervals for each, 95% of these intervals would include the true population proportion of Americans who think it’s the government’s responsibility to promote equality between men and women.**
  - <font color='green'>**TRUE**</font>
    - <font color='blue'>**By the definition of the confidence level. If enough random samples are collected with 95% confidence interval for each, We can be sure that 95% of these intervals contain true population proportion.**</font>

**<font color='orange' size='6px'>D)</font>**  
- **In order to decrease the margin of error to 1%, we would need to quadruple (multiply by 4) the sample size.**
  - <font color='green'>**TRUE**</font>
  - <font color='blue'>**ME = z × $\sqrt{\frac{p̂(p̂-1)}{𝒏}}$.**</font>
  - <font color='blue'>**$\frac{1}{2}$ME = $\frac{1}{2}$z × $\sqrt{\frac{p̂(p̂-1)}{𝒏}}$**</font>
  - <font color='blue'>**$\frac{1}{2}$ME = z × $\sqrt{\frac{p̂(p̂-1)}{4𝒏}}$**</font>
  - <font color='blue'>**Quadrupling the sample size decreases the SE and ME $\frac{1}{\sqrt4}$.**</font>


**<font color='orange' size='6px'>E)</font>**  

- **Based on this confidence interval, there is sufficient evidence to conclude that a majority of Americans think it’s the government’s responsibility to promote equality between men and women.**
  - <font color='green'>**TRUE**</font>
    - <font color='blue'>**The 95% CI is entirely above 50%. On the other hand the lower bound is above 50%.**</font>


**<font color='red' size='8px'>Question 2</font>**

A newspaper collects a simple random sample of 450 likely voters in the district and estimates Toohey's support to be 53%. Does this provide convincing evidence for the claim of Toohey's manager at the 5% significance level?

**<font color='green' size=6px>Answer 👇:**</font>

<font color='blue' size='5px'>**Hypothesis:**</font>
- **$H_0$** : **𝑷** = **0.5**
- **$H_A$** : **𝑷** > **0.5**

<font color='blue' size='5px'>**Check Conditions:**</font>

- <font color='green'>**Independence:**</font> **450 < 10% of voters and all of them are random.**
- <font color='green'>**Sample size/ Skew:**</font> **450 × 0.5 = 225 > 10**
  - <font color='purple'>**S-F condition met →nearly normal sampling distribution.**</font>

Now calculate **SE** :

- **SE** = **$\sqrt{\frac{𝑷 × (𝑷 - 1)}{𝐧}}$**
- **SE** = **$\sqrt{\frac{0.5 × 0.5}{450}}$**
- **SE** = <font color='blue'>**0.023**</font>

Give information:
- **p̂** = **0.53**

Let's get **𝒁**:

- **𝒁** = **$\frac{0.53 - 0.5}{0.023}$** = $\frac{0.03}{0.023}$
- **𝒁** ≈ <font color='blue'>**1.30**</font>

<font color='blue' size='5px'>**Result:**</font>

- **Based on what we calculated, $H_0$ is <font color='red'>rejected.</font> This provides convincing evedence to claim of Toohey's manager.**

**<font color='red' size='8px'>Question 3</font>**

In July 2008, the US National Institutes of Health announced that it was stopping a clinical study early because of unexpected results. The study population consisted of HIV-infected women in sub-Saharan Africa who had been given a single dose of Nevirapine (a treatment for HIV) while giving birth to prevent transmission of HIV to the infant. The study was a randomized comparison of continued treatment of a woman (after successful childbirth) with Nevirapine vs. Lopinavir, a second drug used to treat HIV. 240 women participated in the study; 120 were randomized to each of the two treatments. Twenty-four weeks after starting the study treatment, each woman was tested to determine if the HIV infection was becoming worse (an outcome called virologic failure). 26 of the 120 women treated with Nevirapine experienced virologic failure, while 10 of the 120 women treated with the other drug experienced virologic failure.

- **Create a two-way table presenting the results of this study.**
- **State appropriate hypotheses to test for difference in virologic failure rates between treatment groups.**
- **Complete the hypothesis test and state an appropriate conclusion. (Reminder: Verify any necessary conditions for the test.)**

**<font color='green' size=6px>Answer 👇:**</font>

**<font color='orange' size='6px'>A)</font>**  

<font color='blue' size='5px'></font>| <font color='blue' size='5px'>YES </font>| <font color='blue' size='5px'>NO</font> | <font color='blue' size='5px'>TOTAL </font>
:---:|:---:|:---:|:---:
<font color='blue' size='5px'>**Nevirapine**</font>|<font size='5px'>**26**</font>|<font size='5px'>**94**</font>|<font size='5px'>**120**</font>
<font color='blue' size='5px'>**Lopinavir**</font>|<font size='5px'>**10**</font>|<font size='5px'>**110**</font>|<font size='5px'>**120**</font>
<font color='red' size='5px'>**Total**</font>|<font size='5px'>**36**</font>|<font size='5px'>**204**</font>|<font size='5px'>**240**</font>


**<font color='orange' size='6px'>B)</font>**  

- **$H_0 :$** **$𝑷_N$** = **$𝑷_L$**
  - <font color='blue'>**There is no difference in virologic failure rates between the Nevaripine and Lopinavir groups.**</font>

- **$H_A :$** **$𝑷_N$** $\neq$ **$𝑷_L$**
  - <font color='blue'>**There is no difference in virologic failure rates between the Nevaripine and Lopinavir groups.**



**<font color='orange' size='6px'>C)</font>**  

<font color='green' size='5px'>**Check conditions**:</font>

First of all let's check conditions:
**1. Independence:**

- <font color='blue' >**within groups: random sample and 𝐧 is less than %10**</font><font color='green'>$\surd$</font>
- <font color='blue'>**between groups: no reason to expect this samples are dependent** </font><font color='green'>$\surd$</font>

**1. Sample size / Skew:**

- <font color='blue'>**Lopinavir and Nevirapine groups: 120 × 0.15 = 18 and 120 × 0.85 = 102**.</font><font color='green'>$\surd$</font>


<font color='green' size='5px'>**Complete the hypothesis test**:</font>

First, Let's get **$p̂_{Nevirapine}$** and **$p̂_{Lopinavir}$**:

- **$p̂_{Nevirapine}$:** $\frac{26}{120}$ = <font color='blue'>**0.216**</font>
- **$p̂_{Lopinavir}$:** $\frac{10}{120}$ = <font color='blue'>**0.083**</font>

We have two groups and we want to compare the two proportions. So let's calulate the **$p̂_{pool}$**. The formula for **$p̂_{pool}$** is:

- **$p̂_{pool}$** = **$\frac{26 + 10}{240}$** = <font color='blue'>**0.15**</font>

How about **SE**? Let's get it:

- **SE** = $\sqrt{\frac {p̂_{pool} (1- p̂_{pool})} {𝐧_1} + \frac {p̂_{pool} (1- p̂_{pool})} {𝐧_2}}$

- **SE** = $\sqrt{\frac {0.15 × 0.85} {120} + \frac {0.15 × 0.85} {120}}$

- **SE** = $\sqrt{0.001 + 0.001}$ = $\sqrt{0.002}$

- **SE** = <font color='blue'>**0.044**</font>


Let's get **point estimate**:

- **point estimate** = **$p̂_{Nevirapine}$** - **$p̂_{Lopinavir}$**
- **point estimate** = **0.216 - 0.083**
- **point estimate** = <font color='blue'>**0.133**</font>

Information:
- **Point estimate** = <font color='blue'>**0.133**</font>
- **Null value** = <font color='blue'>**0**</font>
- **SE** = <font color='blue'>**0.044**</font>

So let's calculate **$𝒁$** :

**$𝒁$** = **$\frac{0.133 - 0}{0.044}$** ≈ <font color='blue'>**3.022**</font>


- **p-value** = **𝑃(|𝑍| > 3.022)** ≈ <font color='blue'>**0.0013**</font>

<font color='green' size='5px'>**Result**:</font>

- **Since the p-value is low, we <font color='red'>reject</font> $H_0$ There is strong evidence of a difference in virologic failure rates between the Nevaripine and Lopinavir groups do not appear to be independent.**







**<font color='red' size='8px'>Question 4</font>**

Two different question phrasings are being used to compare the 2010 healthcare law approval. Depending on how a question is phrased, a person may respond differently. For instance, Pew Research Center conducted a survey with the following question:
As you may know, by 2014, nearly all Americans will be required to have health insurance.

          [People who do not buy insurance will pay a penalty]
          while [People who cannot afford it will receive financial help from
          the government]. Do you approve or disapprove of this policy?

For each randomly sampled respondent, the statements in brackets were randomized: either they were kept in the order given above, or the two statements were reversed. The following table shows the results of this experiment. Create and interpret a 90% confidence interval of the difference in approval.


<font color='blue' size='3px'></font>| <font color='blue' size='3px'>Sample size (ni) </font>| <font color='blue' size='3px'>Approve law (%)</font> | <font color='blue' size='3px'>Disapprove law (%) </font>| <font color='blue' size='3px'>Other</font>
:---:|:---:|:---:|:---:|:---:
<font color='darkblue' size='3px'>**People who cannot afford ...**</font>|<font size='3px'>**771**</font>|<font size='3px'>**47**</font>|<font size='3px'>**49**</font>|<font size='3px'>**4**</font>|
<font color='darkblue' size='3px'>**People who do not buy ...**</font>|<font size='3px'>**732**</font>|<font size='3px'>**34**</font>|<font size='3px'>**63**</font>|<font size='3px'>**3**</font>


**<font color='green' size=6px>Answer 👇:**</font>

**<font color='orange' size='6px'>A)</font>**  

Let's denote:

- **p̂_1** :  as the proportion of people approving the law with the statement "People who cannot afford it will receive financial help from the government" is given second.
- **p̂_2** : as the proportion of people approving the law with the statement "People who do not buy it will pay a penalty" is given second.

- **𝐧_1** : as the sample size for the first phrasing.
- **𝐧_2** :as the sample size for the second phrasing.
  


First, Let's get **# of $successes_1$** and **# of $successes_2$** for **$p̂_{pool}$** :

- **# of $successes_1$:** $\frac{47}{100} × 771$ = <font color='blue'>**362.37 ≈ 363**</font>
- **# of $successes_2$:** $\frac{34}{100} × 732$ = <font color='blue'>**248.88 ≈ 249**</font>


Let's get **$p̂_{pool}$**

- **$p̂_{pool}$** = **$\frac{363 + 249}{771 + 732}$** = <font color='blue'>**0.40**</font>

How about **SE**? Let's get it:

- **SE** = $\sqrt{\frac {p̂_{pool} (1- p̂_{pool})} {𝐧_1} + \frac {p̂_{pool} (1- p̂_{pool})} {𝐧_2}}$

- **SE** = $\sqrt{\frac {0.40 × 0.60} {771} + \frac {0.40 × 0.60} {732}}$

- **SE** = $\sqrt{0.00031 + 0.00032}$ = $\sqrt{0.00063}$

- **SE** ≈ <font color='blue'>**0.025**</font>

Calculate **ME**: (based on 𝒁 table for 90% the corresponding value is **1.64**)

- **ME** = **𝒁 × SE**

- **ME** = **1.64 × 0.025**

- **ME** ≈ <font color='blue'>**0.041**</font>


"Create and interpret a 90% confidence interval of the difference in approval".
Let's do it:

- **CL** = **($p̂_1$ - $p̂_2$)** ± **ME**
- **CL** = **($0.47$ - $0.34$)** ± **0.041**
- **CL** = **0.13** ± **0.041**
- **CL** = <font color='blue'>**(0.089 , 0.171)**</font>


<font color='green' size='5px'>**Interpretation:**</font>

- **We are 90% confident that the difference in approval rates for the two question phrasings is between 8.9% and 17.1%. This means that there is a statistically significant difference in approval rates between the two question phrasings, with the statement about financial help from the government being more positively received than the statement about penalties for not buying insurance.**

**<font color='red' size='8px'>Question 5</font>**

Rock-paper-scissors is a hand game played by two or more people where players choose to sign either rock, paper, or scissors with their hands. For your statistics class project, you want to evaluate whether players choose between these three options randomly, or if certain options are favored above others. You ask two friends to play rock-paper-scissors and count the times each option is played. The following table summarizes the data.

Use these data to evaluate whether players choose between these three options randomly, or if certain options are favored above others. Make sure to clearly outline each step of your analysis, and interpret your result in the context of the data and the research question.


 <font color='blue' size='3px'>Rock</font>| <font color='blue' size='3px'>Scissors</font> | <font color='blue' size='3px'>Paper</font>|
:---:|:---:|:---:|
<font size='3px'>**43**</font>|<font size='3px'>**21**</font>|<font size='3px'>**35**</font>



**<font color='green' size=6px>Answer 👇:**</font>

- **$H_0$** = **Each option is equally likely**
- **$H_1$** = **Some options are favored above others**


The **Observed**:

<font color='blue' size='3px'>Rock</font>| <font color='blue' size='3px'>Scissors</font> | <font color='blue' size='3px'>Paper</font>|
:---:|:---:|:---:|
<font size='3px'>**43**</font>|<font size='3px'>**21**</font>|<font size='3px'>**35**</font>

The **Expected**:

Each option has equally chance so the total number is:

- **T** = **43 + 21 + 35** = <font color='blue'>**99**</font>

- **Expected counts** = $\frac{1}{3}$ × 99 = <font color='blue'>**33**</font> (for each option)

So the **Expected** table is:

<font color='blue' size='3px'>Rock</font>| <font color='blue' size='3px'>Scissors</font> | <font color='blue' size='3px'>Paper</font>|
:---:|:---:|:---:|
<font size='3px'>**33**</font>|<font size='3px'>**33**</font>|<font size='3px'>**33**</font>

<font color='green' size='5px'>**Check conditions:**</font>

<font color='blue' size='3px'>**Independence:**</font>
- **Sampled observations must be independent**<font color='green'>$\surd$</font>
- **99 < 10%**<font color='green'>$\surd$</font>
- **Random sample/assignment**<font color='green'>$\surd$</font>
- **each case only contributes to one cell in the table**<font color='green'>$\surd$</font>

<font color='blue' size='3px'>**Sample size:**</font>
- **Each particular scenario (i.e.cell) must have at least 5 expected cases**<font color='green'>$\surd$</font>



Now it's time to calculate **Chi-square ($𝓧^2$) statistic**:

- **$𝓧^2$** = $\sum_{𝑖=1}^k \frac{(𝑂 - 𝐸)^2}{𝐸}$

- **$𝓧^2$** = $\frac{(43 - 33)^2}{33} + \frac{(21 - 33)^2}{33} + \frac{(35 - 33)^2}{33}$

- **$𝓧^2$** = $\frac{(10)^2}{33} + \frac{(-12)^2}{33} + \frac{(2)^2}{33}$
- **$𝓧^2$** = $\frac{100}{33} + \frac{144}{33} + \frac{4}{33}$

- **$𝓧^2$** = **3.03 + 4.36 + 0.12** = <font color='blue'>**7.51**</font>

How about **degrees of freedom**?

- **df** = **3 - 1** = <font color='blue'>**2**</font>

Based on **Chi square** table 7.51 is:

- 0.01 < <font color='purple'>p-value</font> < 0.025

<font color='green' size='5px'>**Result:**</font>


- **Since the p-value is less than 5%, we <font color='red'>reject</font> $H_0$. The data provide convincing evidence that some options are preferred over others.**

**<font color='red' size='8px'>Question 6</font>**

A Restaurant Manager for a Football Park wants to know if there is a relationship between gender (male or female) and the preferred condiment on a sandwich. The following table summarizes the results. Test the hypothesis with a significance level of 5%.


<font color='blue' size='3px'></font>| <font color='blue' size='3px'>Ketchup</font>| <font color='blue' size='3px'>Mustard</font> | <font color='blue' size='3px'>Relish</font>| <font color='blue' size='3px'>Total</font>
:---:|:---:|:---:|:---:|:---:
<font color='darkblue' size='3px'>**Male**</font>|<font size='3px'>**15**</font>|<font size='3px'>**23**</font>|<font size='3px'>**10**</font>|<font size='3px'>**48**</font>|
<font color='darkblue' size='3px'>**Female**</font>|<font size='3px'>**25**</font>|<font size='3px'>**19**</font>|<font size='3px'>**8**</font>|<font size='3px'>**52**</font>|
<font color='darkblue' size='3px'>**Total**</font>|<font size='3px'>**40**</font>|<font size='3px'>**42**</font>|<font size='3px'>**18**</font>|<font size='3px'>**100**</font>


**<font color='green' size=6px>Answer 👇:**</font>

It is solved by **Chi-square** independence test.

<font color='green' size='5px'>**Hypothesis test:**</font>
- **$H_0$** = **Gender and condiments are independent.**

- **$H_A$** = **Gender and condiments are not independent.**

First We need to get **Expected** counts:


- **$E_{row_1, col_1}$** = **$\frac{48 × 40}{100}$** = <font color='blue'>**19.2**</font>
- **$E_{row_2, col_1}$** = **$\frac{52 × 40}{100}$** = <font color='blue'>**20.8**</font>
- **$E_{row_1, col_2}$** = **$\frac{48 × 42}{100}$** = <font color='blue'>**20.16**</font>
- **$E_{row_2, col_2}$** = **$\frac{52 × 42}{100}$** = <font color='blue'>**21.84**</font>
- **$E_{row_3, col_1}$** = **$\frac{48 × 18}{100}$** = <font color='blue'>**8.64**</font>
- **$E_{row_3, col_2}$** = **$\frac{52 × 18}{100}$** = <font color='blue'>**9.36**</font>


<font color='blue' size='3px'></font>| <font color='blue' size='3px'>Ketchup</font>| <font color='blue' size='3px'>Mustard</font> | <font color='blue' size='3px'>Relish</font>| <font color='blue' size='3px'>Total</font>
:---:|:---:|:---:|:---:|:---:
<font color='darkblue' size='3px'>**Male**</font>|<font size='3px'>**15 (19.2)**</font>|<font size='3px'>**23 (20.16)**</font>|<font size='3px'>**10 (8.64)**</font>|<font size='3px'>**48**</font>|
<font color='darkblue' size='3px'>**Female**</font>|<font size='3px'>**25 (20.8)**</font>|<font size='3px'>**19 (21.84)**</font>|<font size='3px'>**8 (9.36)**</font>|<font size='3px'>**52**</font>|
<font color='darkblue' size='3px'>**Total**</font>|<font size='3px'>**40**</font>|<font size='3px'>**42**</font>|<font size='3px'>**18**</font>|<font size='3px'>**100**</font>


Now it's time to calculate **Chi-square ($𝓧^2$) statistic**:

- **$𝓧^2$** = $\sum_{𝑖=1}^k \frac{(𝑂 - 𝐸)^2}{𝐸}$

- **$𝓧^2$** = $\frac{(15 - 19.2)^2}{19.2} + \frac{(25 - 20.8)^2}{20.8} + \frac{(23 - 20.16)^2}{20.16} + \frac{(19 - 21.84)^2}{21.84} + \frac{(10 - 8.64)^2}{8.64} + \frac{(8 - 9.36)^2}{9.36}$

- **$𝓧^2$** = $\frac{(-4.2)^2}{19.2} + \frac{(4.2)^2}{20.8} + \frac{(2.84)^2}{20.16} + \frac{(-2.84)^2}{21.84} + \frac{(1.36)^2}{8.64} + \frac{(-1.36)^2}{9.36}$
- **$𝓧^2$** = $\frac{17.64}{19.2} + \frac{17.64}{20.8} + \frac{8.06}{20.16} + \frac{8.06}{21.84} + \frac{1.84}{8.64} + \frac{1.84}{9.36}$

- **$𝓧^2$** = **0.918 + 0.848 + 0.399 + 0.369 + 0.212 + 0.196** = <font color='blue'>**2.94**</font>

How about **degrees of freedom**?

- **df** = **(𝘙 - 1) × (𝐶 - 1)**

We have **3** columns and **2** rows, S0:

- **df** = **(2 - 1) × (3 - 1)** = **(1) × (2)** = <font color='blue'>**2**</font>

<font color='purple' size='5px'>**p-value:**</font>
- **The p-value for 𝑃($𝓧^2$ > 2.94) with <font color='blue'>2</font> degrees of freedom is <font color='green'>0.2299</font>**

> <font color='red'>**Note:**</font>
- **The p value is obtained by r (code below)**

<font color='green' size='5px'>**Result:**</font>

- **p-value <font color='green'>0.2299 > 0.05%</font>, We fail to <font color='red'>reject</font> $H_0$. We can conclude that there is <font color='red'>not</font> enough evidence in the data to suggest that gender and preferred condiment are related.**

In [ ]:
# getting p-value for Chi-square
pchisq(2.94, 2,lower.tail =FALSE)

[1] 0.2299255

**<font color='red' size='8px'>Question 7</font>**

During a five-day workweek, employers are interested in knowing which days employees are absent. It is usually believed by employers that employees are absent equally throughout the week. In this scenario, a random sample of 70 managers is asked what day of the week they had the highest number of employee absences. The results were distributed as in Table 7.1. For the population of employees, do the days for the highest number of absences occur with equal frequencies during a five-day workweek? Test at a 5% significance level. (Using a chi-square test)


<font color='blue' size='3px'></font>| <font color='blue' size='3px'>Monday</font>| <font color='blue' size='3px'>Tuesday</font> | <font color='blue' size='3px'>Wednesday</font>| <font color='blue' size='3px'>Thursday</font>| <font color='blue' size='3px'>Friday</font>
:---:|:---:|:---:|:---:|:---:|:---:
<font color='darkblue' size='3px'>**Number of Absences**</font>|<font size='3px'>**18**</font>|<font size='3px'>**16**</font>|<font size='3px'>**10**</font>|<font size='3px'>**10**</font>|<font size='3px'>**16**</font>


**<font color='green' size=6px>Answer 👇:**</font>

<font color='navy' size='5px'>**The null and alternative hypotheses are:**</font>

- **$H_0$** = **The absent days occur with equal frequencies.**
- **$H_A$** = **The absent days occur with unequal frequencies.**

In [ ]:
# Create observed table like above
observed_table <- matrix(c(18, 16, 10, 10, 16), nrow = 1, ncol = 5, byrow = T)
rownames(observed_table) <- c('Number of Absences')
colnames(observed_table) <- c('Monday', 'Tuesday', 'Wednesday','Thursday','Friday')
observed_table

Monday Tuesday Wednesday Thursday Friday
Number of Absences 18     16      10        10       16

In [ ]:
X <- chisq.test(observed_table)
X


	Chi-squared test for given probabilities

data:  observed_table
X-squared = 4, df = 4, p-value = 0.406


<font color='green' size='5px'>**Result:**</font>

- **p-value 0.406 > 0.05, So We fail to <font color='red'>reject</font> null hypothesis ($H_0$).**

**<font color='red' size='8px'>Question 8</font>**

Consider the dataset named “survey” in the “MASS” library in R. The Smoke column records the students’ smoking habit, while the “Exer” column records their exercise level. Test the hypothesis of whether the students’ smoking habit is independent of their exercise level at the 0.05 significance level.



**<font color='green' size=6px>Answer 👇:**</font>

<font color='navy' size='5px'>**The null and alternative hypotheses are:**</font>

- **$H_0$** = **Smoking habit and exercise level are independent.**
- **$H_A$** = **Smoking habit and exercise level are dependent.**

In [ ]:
library(MASS) # load the MASS package

In [ ]:
data("survey") # load data

In [ ]:
# Creating table with Smoke and Exer columns
table_data  <- table(survey$Smoke , survey$Exer)
table_data

       
        Freq None Some
  Heavy    7    1    3
  Never   87   18   84
  Occas   12    3    4
  Regul    9    1    7

In [ ]:
# # Perform the Chi-squared test of independence
X <- chisq.test(df)
X

Warning message in chisq.test(df):
“Chi-squared approximation may be incorrect”



	Pearson's Chi-squared test

data:  df
X-squared = 5.4885, df = 6, p-value = 0.4828


In [ ]:
# Check the p-value
if(X$p.value < 0.05) {
  print("Reject the null hypothesis: Smoking habit and exercise level are dependent.")
} else {
  print("Fail to reject the null hypothesis: Smoking habit and exercise level are independent.")
}

[1] "Fail to reject the null hypothesis: Smoking habit and exercise level are independent."


<font color='green' size='5px'>**Result:**</font>

- **p-value 0.4828 > 0.05, So We fail to <font color='red'>reject</font> null hypothesis ($H_0$).**